Formula/r/runitor.rb

```diff
@@ -0,0 +1,22 @@
+class Runitor < Formula
+  desc "Command runner with healthchecks.io integration"
+  homepage "https://github.com/bdd/runitor"
+  url "https://github.com/bdd/runitor/archive/refs/tags/v1.3.0-build.4.tar.gz"
+  version "1.3.0-build.4"
+  sha256 "0dd2768ecfaeac14dda2e587d2ecc3cf28672e7ca86d117edb8633303144bc2b"
+  license "0BSD"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}"), "./cmd/runitor"
+  end
+
+  test do
+    assert_match(/Ping\(start\): nonretriable error response: 400 Bad Request/,
+      shell_output(bin/"runitor -uuid 00000000-0000-0000-0000-000000000000 true 2>&1"))
+
+    assert_equal("runitor #{version}",
+      shell_output(bin/"runitor -version").strip)
```

Comment:
```suggestion
  output = shell_output("#{bin}/runitor -uuid 00000000-0000-0000-0000-000000000000 true 2>&1")
  assert_match "error response: 400 Bad Request/", output

  assert_equal version.to_s, shell_output("#{bin}/runitor -version")
```

Reply:
Amended with those changes in spirit, that syntax is not quite right

Reply:
oh, i was thinking of `assert_match version.to_s, shell_output("#{bin}/runitor -version")`, either way, you go it. 👍